coldfront/core/project/templates/project/project_add_users.html

```diff
@@ -29,7 +29,9 @@ <h1>Add users to project: {{project.title}}</h1> <hr>
 <div class="col" id="search_results" >
     <div class="card border-light">
       <div class="card-body">
-        <p>You may only add users who have signed the access agreement (no check box will appear below for those who have not signed). In addition, users who have requests to join this project appear <a href="{% url 'project-review-join-requests' project.pk %}">here</a>.</p>
+          <div class="alert alert-warning" role="alert">
+              <p>You may only add users who have signed the access agreement (no check box will appear below for those who have not signed). In addition, users who have requests to join this project appear <a href="{% url 'project-review-join-requests' project.pk %}">here</a>.</p>
```

Comment:
I'm seeing an extra newline at the end of this alert, probably caused by the `<p>` within the `<div>`.

![spacing](https://user-images.githubusercontent.com/10734545/134255598-63b675ac-9bc4-41d4-a0a6-061000e70a9e.png)